FusionIIIT/templates/ais/ais.html

```diff
@@ -0,0 +1,209 @@
+
+{% extends 'fusion/general/base.html' %}
```

Comment:
Change it accordingly. 
These URL's won't work!
Have a look at the code I pushed.
ASAP!